apps/docs/src/pages/components/avatar.mdx

```diff
@@ -1,7 +1,7 @@
 import { avatarClasses, HvAvatar } from "@hitachivantara/uikit-react-core";
 
-import { Description } from "@docs/components/Description";
-import { Page } from "@docs/components/Page";
+import { Description } from "@docs/components/component/Description";
+import { Page } from "@docs/components/component/Page";
```

Comment:
If these are only for components, maybe we could simplify and just leave on the components page (like `Playground` is) instead of adding more nesting

```suggestion
import { Description } from "./Description";
import { Page } from "./Page";
```

Reply:
We can take this offline. Personally i prefer to keep this separation.